dolphinscheduler-worker/src/main/java/org/apache/dolphinscheduler/server/worker/utils/TaskExecutionContextUtils.java

```diff
@@ -51,6 +52,9 @@ public static void createTaskInstanceWorkingDirectory(TaskExecutionContext taskE
                         taskInstanceWorkingDirectory);
             }
             FileUtils.createDirectoryWith755(Paths.get(taskInstanceWorkingDirectory));
+            final Path taskInstanceWorkingDirectoryPath = Paths.get(taskInstanceWorkingDirectory);
```

Comment:
If we change to `775` and make bootstrap user has the same group with tenant, then tenant will have permission to create file.

Reply:
> If we change to `775` and make bootstrap user has the same group with tenant, then tenant will have permission to create file at working directory.

So, can i create a function createDirectoryWith775() and call it here ?

Reply:
I think we can. cc @caishunfeng, @SbloodyS

Reply:
I'm ok with this. If we change the task instance working directory to 775, we should add it to the installation docs to tell users.

Reply:
@SbloodyS 
Which part of the document should I update?
Also, please review the code i just committed.

Thanks.

Reply:
https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/zh/guide/security/security.md

Reply:
> https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/zh/guide/security/security.md https://github.com/apache/dolphinscheduler/blob/dev/docs/docs/en/guide/security/security.md
> 
> The create tenant part.

Finished.